messaging/pom.xml

```diff
@@ -121,4 +144,33 @@
 			<scope>test</scope>
 		</dependency>
 	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-jar-plugin</artifactId>
+				<configuration>
+					<archive>
+						<addMavenDescriptor>true</addMavenDescriptor>
+						<manifest>
+							<addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+							<addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+							<mainClass>org.frankframework.cmdline.StartIbis</mainClass>
+						</manifest>
+						<manifestEntries>
+							<Add-Opens>java.base/java.lang java.base/java.io</Add-Opens>
+						</manifestEntries>
+					</archive>
+				</configuration>
```

Comment:
```suggestion
```
dit kan hij in heriten, als je puur de test jar nodig hebt

Reply:
Ik moet een nieuwe test-jar maken, omdat hier nu een paar JMS test classes inzitten die nodig zijn voor EsbJmsListener in NN-Specials.
De main-class is natuurlijk niet nodig... De rest misschien ook niet, maar daar had ik nog niet zo over nagedacht!